src/super_gradients/examples/qat_from_recipe_example/qat_from_recipe.py

```diff
@@ -0,0 +1,239 @@
+"""
+Code for running PTQ/QAT on SuperGradients recipes.
+
+THIS SCRIPT WILL MODIFY YOUR RECIPE TO SUIT QAT.
+To use recipe as is, set `quantization_params.modify_recipe_for_qat.enable` to False
+
+This script is proven NOT to work with DDP and will disable it automatically!
+
+if `training_hyperparams.max_epochs=0`, only PTQ will be performed!
+
+Usage:
+    python qat_from_recipe.py
+        --config-name=your_recipe
+        +quantization_params=default_quantization_params OR your_desired_quantization_params
+        +checkpoint_params.checkpoint_path=/full/path/to/your/checkpoint
+"""
+
+import os
+
+import hydra
+import pkg_resources
+from omegaconf import DictConfig
+from torch import nn
+
+from super_gradients import Trainer, init_trainer
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.common.data_types.enum import MultiGPUMode
+from super_gradients.training import utils as core_utils, models, dataloaders
+from super_gradients.training.metrics.metric_utils import get_metrics_dict
+from super_gradients.training.utils import get_param
+from super_gradients.training.utils.distributed_training_utils import setup_device
+from super_gradients.training.utils.module_utils import fuse_repvgg_blocks_residual_branches
+from super_gradients.training.utils.quantization.calibrator import QuantizationCalibrator
+from super_gradients.training.utils.quantization.export import export_quantized_module_to_onnx
+from super_gradients.training.utils.quantization.selective_quantization_utils import SelectiveQuantizer
+
+logger = get_logger(__name__)
+
+
+def quantize_and_calibrate(
```

Comment:
Lets move this to a more appropriate place (see summary).
Also, please add docstring.

Reply:
Can we also separate this into calibrate and quantise ?